mvrx-navigation/src/main/kotlin/com/airbnb/mvrx/navigation/MvRxNavDirections.kt

```diff
@@ -0,0 +1,47 @@
+package com.airbnb.mvrx.navigation
+
+import android.os.Bundle
+import android.os.Parcelable
+import androidx.annotation.IdRes
+import androidx.core.os.bundleOf
+import androidx.navigation.NavDirections
+import com.airbnb.mvrx.Mavericks
+
+/**
+ * @param actionId - the navigation graph action ID.
+ * @param data -  the a Parcelable MvRx that is passed as the [Mavericks.KEY_ARG] fragment argument
+ * @param title - an optional title that the destination can extract in the graph definition
+ *
+ * [title] usage examples
+ *
+ * ```xml
+ *   <fragment
+ *      android:id="@+id/financialAccountDetailsFragment"
+ *      android:name="example.app.UserSettingsFragment"
+ *      android:label="{mvrx:arg:title}">
+ *
+ *       <argument
+ *          android:name="mvrx:arg:title"
+ *          app:argType="string" />
+ *
+ *   </fragment>
+ * ```
+ */
+class MvRxNavDirections(
+    @IdRes private val actionId: Int,
+    private val data: Parcelable,
```

Comment:
I think you want a version of this for Serializable too

Reply:
Ah, yes, I forgot Nav args can be Serializable.

Reply:
Updated using a create to pick between `Parcelable` or `Serializable` data types. Let me know what you think.